README.md

```diff
@@ -173,8 +173,6 @@ This method will create a configured client on the native modules. Resolves `tru
 
 **Note**: `androidChromeTabColor` is an optional field in config, and is used only by Android for the Chrome Custom Tabs color for the OIDC flow.
 
-**Note**: `browserMatchAll` is an optional field in config, and is used only by Android to match all Chrome Custom Tabs browsers.
```

Comment:
I'm confused, how did this change get into the 1.9 branch?  If it needs to be reverted, please do that in a separate PR to keep the changes distinct.

Reply:
It is outdated changes. Please, take a look at `Files` tab.